packages/fxa-settings/src/pages/Signin/SigninUnblock/index.test.tsx

```diff
@@ -114,7 +114,7 @@ describe('SigninUnblock', () => {
     renderWithSuccess();
 
     screen.getByRole('heading', { name: 'Authorize this sign-in' });
-    screen.getByRole('img', { name: 'An envelope containing a link' });
+    screen.getByRole('img', { name: 'Illustration to represent an email containing a code.' });
```

Comment:
Do we want to move away from "Illustration"?

Reply:
We do, and my PR was initially even bigger because I had also updated all the image aria-labels... I opted to back out those changes after rebasing off the other two recent PRs that updated images/icons. I can file a ticket.

The reason for the switch here is that I replaced the image used on this page (we still had the email link image even though this an email code) but did not make changes to the images themselves.